tests/test_tools.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from pkg_resources import resource_filename
+from collections import namedtuple
+from wps_tools.utils import (
+    is_opendap_url,
+    get_filepaths,
+    collect_output_files,
+    build_meta_link,
+)
+from wps_tools.testing import (
+    local_path,
+    opendap_path,
+    run_wps_process,
+)
+from .processes.wps_test_process import TestProcess
+
+NCInput = namedtuple("NCInput", ["url", "file"])
+NCInput.__new__.__defaults__ = ("", "")
+
+nc_file = "gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100.nc"
+
+
+# Test 'testing' functions
```

Comment:
We want to separate tests by their module. This ensures we have the flexibility to target specific module tests during development and in CI.